ImperatorToCk3/Source/Configuration/ConfigurationDetails.h

```diff
@@ -0,0 +1,27 @@
+#ifndef CONFIGURATION_DETAILS_H
+#define CONFIGURATION_DETAILS_H
+
+#include "Date.h"
+#include "newParser.h"
+
+class ConfigurationDetails: commonItems::parser
+{
+  public:
+	ConfigurationDetails() = default;
+	explicit ConfigurationDetails(std::istream& theStream);
+
+	enum class IMPERATOR_DE_JURE { PROVS_AND_REGIONS = 1, COUNTRIES = 2, NO = 3 };
```

Comment:
This is fine, just keep in mind the compiler won't allow you to have two enum classes with same "NO" key. Using specific "NO_JURE" or similar will avoid this issue.

Reply:
I don't think that's correct. One of the advantages of scoped enums (`enum class`) is that they can have keys with the same names, because you have to scope into them anyways.

Reply:
Ah, my bad, I mistook that with another instance of regular enum. All is well.